runtime-sdk/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = "Apache-2.0"
 [dependencies]
 # TODO: Use specific tag.
 oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", branch = "master" }
+runtime-sdk-macros = { path = "../runtime-sdk-macros", optional = true }
```

Comment:
This should probably be on by default?

Reply:
That makes sense. I was thinking that maybe people don't want to pull in `syn` for a little macro, but I guess they already are!